BlueCombing/Views/MakeCardViews/MakeCardView.swift

```diff
@@ -20,132 +20,145 @@ struct MakeCardView: View {
-            
-            Spacer()
-            Button(action: {
-                let saveImage = SaveImageView.snapshot()
-                presentBadge = userViewModel.user!.representBadge
-                isPresentShareView.toggle()
-                userViewModel.uploadPicture(image: saveImage)
-                userViewModel.updateUser(uid: userViewModel.user!.id, info: ["total_time": userViewModel.user!.totalTime + movingTime, "total_distance":userViewModel.user!.totalDistance+Int(movingDistance)])
-            }){
-                ZStack {
-                    Rectangle()
-                        .fill(Color.combingGradient1)
-                        .frame(width: containerWidth, height: 56)
-                        .cornerRadius(16)
-                    Text("완료")
-                        .font(.system(size: 16, weight: .semibold))
-                        .foregroundColor(.white)
-                }
+                completeButton
+
             }
+            .frame(width: containerWidth)
             .navigationTitle("")
             .navigationBarHidden(true)
-        }
-        .sheet(isPresented: $isPresentedAllImage){
-            AllImagePicker(cardViewModel: cardViewModel, card: $cardViewModel.newCard)
-        }
-        .sheet(isPresented: $isPresentedLimitedImage){
-            LimitedImagePicker(cardViewModel: cardViewModel, card: $cardViewModel.newCard)
-        }
-        .sheet(isPresented: $isPresentedCamera) {
-            Camera(cardViewModel: cardViewModel, card: $cardViewModel.newCard)
-        }
-        .alert("이 기능을 사용하려면 카메라 엑세스 권한이 필요합니다", isPresented: $cameraDenyAlert) {
-            Button("나중에 하기") {
-                
+            .sheet(isPresented: $isPresentedAllImage){
+                AllImagePicker(cardViewModel: cardViewModel, card: $cardViewModel.newCard)
             }
-            Button("설정 열기") {
-                if let settingsURL = URL(string: UIApplication.openSettingsURLString) {
-                    UIApplication.shared.open(settingsURL, options: [:], completionHandler: nil)
-                }
+            .sheet(isPresented: $isPresentedLimitedImage){
+                LimitedImagePicker(cardViewModel: cardViewModel, card: $cardViewModel.newCard)
             }
-        }.sheet(isPresented: $isPresentShareView){
-            ShareView(card: $cardViewModel.newCard, presentBadge: $presentBadge, routeImage: routeImage) {
-                dismiss()
+            .sheet(isPresented: $isPresentedCamera) {
+                Camera(cardViewModel: cardViewModel, card: $cardViewModel.newCard)
             }
-        }.onAppear {
-            if firebaseAuth.currentUser != nil {
-                isSignin = true
-                userViewModel.getUserData(uid: firebaseAuth.currentUser!.uid)
-            } else {
-                isSignin = false
+            .alert("이 기능을 사용하려면 카메라 엑세스 권한이 필요합니다", isPresented: $cameraDenyAlert) {
+                Button("나중에 하기") {
+                    
+                }
```

Comment:
빈 클로져는 한 줄로 작성이 좋아보입니다.
```suggestion
                Button("나중에 하기") {  }
```

Reply:
수정했습니다!!